Project/Assets/Editor/CustomInspectors/SlideshowEditor.cs

```diff
@@ -21,6 +28,8 @@ public override void OnInspectorGUI()
 			}
 		}
 
+        slideshow.Slides = PrairieGUI.RemoveNulls(editorSlides);
```

Comment:
We have a method in this editor `DrawWarnings()`, which is run if one or more slides is set to `null`, and warns the user that they should populate the array.

With this PR, we remove the possibility that any value in a component's array is set to null at any time. So that warning probably needs to go.